proto/composerapis/astria/composer/v1alpha1/grpc_collector.proto

```diff
@@ -0,0 +1,30 @@
+syntax = 'proto3';
+
+package astria.composer.v1alpha1;
+
+// RollupTransaction is a message that represents a single rollup transaction that is to be submitted to the
+// Shared Sequencer Network via the Composer
+message RollupTransaction {
+  // the unhashed rollup id
+  bytes rollup_id = 1;
+  // the raw data bytes of the rollup transaction
+  bytes data = 2;
+}
+
+// SubmitRollupTransactionsRequest contains a batch of rollup transactions to be submitted to the Shared Sequencer
+// Network via the Composer
+message SubmitRollupTransactionsRequest {
+  // an array of rollup transactions to be submitted
+  repeated RollupTransaction rollup_transactions = 1;
```

Comment:
Should not be a `repeated` - keeping a single tx per request will make tracing easier. We end up breaking these apart into bundles inside `composer` anyways, so keeping it at one tx per request would allow rollup nodes to attach an id to the request that would identify the tx.

Reply:
makes sense. I think we would have to understand how to define the id. But I think keeping it at 1 tx should be pretty good.